### PR TITLE
Change PR template

### PR DESCRIPTION
**Pull request by simonchenwastaken**
## Motivation and Context
Making the file more consistent with syntax

## Your Changes
- Refactor Markdown HTML comment tags to be consistent within the file. Previously, there were both `<!--` and `<!---`.

**Description**:

**Type of change** (select all that apply):

 

- [x] Refactoring (internal change to codebase, without changing functionality)

## Testing

N/A

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have verified that the TravisCI tests have passed. 
- [x] I have reviewed the test coverage changes reported on Coveralls.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+<!-- Provide a summary of your changes in the Pull Request Title above. -->
+<!-- If this is a work in progress (not yet ready to be merged), make this a draft pull request. -->
 
 ## Motivation and Context
 
-<!--- Why is this pull request required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to the issue here: -->
-<!--- https://docs.github.com/en/github/managing-your-work-on-github/managing-your-work-with-issues-and-pull-requests/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword -->
+<!-- Why is this pull request required? What problem does it solve? -->
+<!-- If it fixes an open issue, link the issue by typing in 'resolves #' then the issue number like (resolves #709) -->
```
Comment (david-yz-liu) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: I deliberately linked the documentation so that you would read it. Revert this change.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 
 ## Your Changes
 
-<!--- Describe your changes here. -->
 **Description**:
+<!-- Describe your changes here. -->
```
Comment (david-yz-liu) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: The heading is fine where it is. Revert this change.

**Review by david-yz-liu — COMMENTED** [review]
@simonchenwastaken I agreed with one of your proposed changes and disagreed with two of them. Please update your PR accordingly.

**Comment by david-yz-liu** [comment]
@simonchenwastaken great, also please update the PR description accordingly.

**Comment by simonchenwastaken** [comment]
> @simonchenwastaken great, also please update the PR description accordingly.

I have updated it to hopefully be more accurate with the new changes!
